src/pages/signin/ResendValidationForm.js

```diff
@@ -99,8 +99,8 @@ class ResendValidationForm extends React.Component {
                 login,
                 loginType,
             });
-        } else if (isOldUnvalidatedAccount) {
-            message = this.props.translate('resendValidationForm.unvalidatedAccount');
```

Comment:
`resendValidationForm.unvalidatedAccount` looks unused now?

Reply:
We need to remove the code in these two locations:

https://github.com/Expensify/App/blob/0c9187d809c4cbbbb44b038b80b6aa650221b275/src/languages/en.js#L408
https://github.com/Expensify/App/blob/0c9187d809c4cbbbb44b038b80b6aa650221b275/src/languages/es.js#L408

Reply:
Please do this.

Reply:
or we are using it now? what was the original design that was agreed on... confused...

Reply:
We're using it now for the condition where the account exists. So that we split the account unvalidated and validation code expired conditions